config/log4j2.properties

```diff
@@ -0,0 +1,36 @@
+status = error
+name = LogstashPropertiesConfig
+
+property.filename = /tmp/logstash/logstash.log
```

Comment:
I would gladly discuss this after this PR makes it in.

It may make more sense for us to provide a very "dumb" config as a starter, and this may be done best in a `.properties` format. Likewise, it will keep consistency with the `.properties` that Elasticsearch will have in 5.0.

Since this is a generic config, I will make sure you are free to use your own custom `.xml` config